client/src/Journal.css

```diff
@@ -0,0 +1,60 @@
+.container {
+  max-width: 575px;
+  margin: 0 auto;
+  padding: 0 1.5rem;
+}
+
+.header {
```

Comment:
FYI 💡: I would use the element `header` instead of class because with the current code structure, it seems a bit redundant. 

However,  in a larger app, this could be the better approach!